### DISCUSSION TITLE
Socketioxide got listed on the socket.io website! 🤩

### DISCUSSION THREAD

**Discussion by tausifcreates** [Totodore/socketioxide#99] [General]
Link - https://socket.io/docs/v4/#server-implementations
![sktio](https://github.com/Totodore/socketioxide/assets/93582436/f9a3a41b-ac8a-4a79-84bc-7e666c148341)

**Comment by Totodore** [comment]
It's coming from here : https://github.com/socketio/socket.io-protocol/pull/31#issuecomment-1726207328

**Reply by tausifcreates** [reply]
The Socketio maintainer is nice. He is pushing to make it more accessible on all platforms.
